src/utils/billing-utils.ts

```diff
@@ -1,8 +1,8 @@
 import { parseISO } from 'date-fns';
 import prettyBytes from 'pretty-bytes';
-import { CatalogStats_Billing, Entity } from 'types';
+import { CatalogStats_Billing, Entity, Schema } from 'types';
 
-export const TOTAL_CARD_HEIGHT = 300;
+export const TOTAL_CARD_HEIGHT = 450;
```

Comment:
Why was this height increased? Originally, Dave wanted as much of the _Billing_ page content to be seen on an "average" laptop screen as possible. The charts are too tall to me at this height. I arrived at the previous height because it neatly fit the contents of the recent billing history table.

<img width="960" alt="graph" src="https://github.com/estuary/ui/assets/77648584/79624b6b-69e4-435c-94e8-b1383cdf8cd5">

If Dave is comfortable with the new height, I would at least extend the recent history table to show the last six months worth of billing history. Otherwise the card will have a considerable amount of empty space beneath the table.

Reply:
Ooof... that's a mistake - I was jacking with heights to test out resizer. I'll put that back.